### DISCUSSION TITLE
Built-in search tags layout

### DISCUSSION THREAD

**Discussion by Vanja-S** [fuma-nama/fumadocs#1138] [Q&A] [answered]
Hi!

I have set up tag filtering search using the built-in search provider.

```tsx
export default function Layout({ children }: { children: ReactNode }) {
  return (
    <html lang="en" className={mulish.className} suppressHydrationWarning>
      <body className="flex flex-col min-h-screen">
        <RootProvider
          search={{
            options: {
              defaultTag: '',
              tags: source.pageTree.children.map((item) => {
                const folderItem = item as Folder;
                return {
                  name: folderItem.name?.toString() || '',
                  value: folderItem.name?.toString().toLowerCase().replaceAll(' ', '-') || ''
                };
              }),
            }
          }}>
          {children}
        </RootProvider>
      </body>
    </html>
  );
}
```

But I have one problem, the tags overflow and don't wrap:
![image](https://github.com/user-attachments/assets/e137f44b-4c05-4b8b-9156-f33e675eb675)

Can you specify this CSS manually without creating a new component in the RootProvider?
I can also try to open a MR request to include this in the built-in search if needed :)

**Comment by fuma-nama** [comment]
Oh needing to wrap the items is definitely a bad UX, it shouldn't have more than 5 items, that is too crowded for a search dialog. You could replace it with a custom footer and use a select/dropdown component for filters instead.

See https://fumadocs.vercel.app/docs/ui/search#replace-search-dialog

**Reply by Vanja-S** [reply, marked as answer]
Hey, got it, thanks for the advice, I will probably implement a different solution then :)
